testing/smoke-ide-test/src/smokeIdeTest/groovy/org/gradle/ide/sync/AbstractIdeSyncTest.groovy

```diff
@@ -51,18 +53,31 @@ abstract class AbstractIdeaSyncTest extends Specification {
         return IntegrationTestBuildContext.INSTANCE
     }
 
+    /**
+     * Runs a full sync process for the build-under-test with a given Android Studio version.
+     * <p>
+     */
+    protected void androidStudioSync(String version) {
+        ideSync("ai-$version")
+    }
+
     /**
      * Runs a full sync process for the build-under-test with a given IntelliJ IDEA Community version.
      * <p>
      * The version can be optionally suffixed with a "build type", which is one of {@code release}, {@code rc}, {@code eap}.
      * For instance, {@code 2024.2-eap}. When the build type is not provided, it defaults to {@code release}.
      * <p>
-     * The sync runs as an external process.
-     * The IDE distribution is automatically downloaded if required.
      */
     protected void ideaSync(String version) {
+        ideSync("ic-$version")
+    }
+
+    /**
+     * Runs a full sync with a given IDE as an external process.
+     * The IDE distribution is automatically downloaded if required.
+     */
```

Comment:
💅 

Could document the parameter stating expected format/supported variants.

Reply:
I intentionally didn't do that, because then it will be a duplication of `gradle-ide-starter` documentation. I'm trying to hide these implementation details, making `ideSync` method private and expose two high-level methods for running a sync, which are self-explanatory, I hope.